frappe/model/document.py

```diff
@@ -723,6 +726,44 @@ def _evaluate_alert(alert):
 			elif alert.event=='Method' and method == alert.method:
 				_evaluate_alert(alert)
 
+	def run_webhooks(self, method):
+		'''Run webhooks for this method'''
+		if frappe.flags.in_import or frappe.flags.in_patch or frappe.flags.in_install:
```

Comment:
lets move this code to web_hooks. I know its here in email_alert, but other wise `document.py` will just keep on increasing...